packages/core/src/Stores/traders-hub-store.js

```diff
@@ -812,36 +809,8 @@ export default class TradersHubStore extends BaseStore {
         this.is_account_transfer_modal_open = !this.is_account_transfer_modal_open;
     }
 
-    toggleFailedVerificationModalVisibility() {
-        this.is_failed_verification_modal_visible = !this.is_failed_verification_modal_visible;
-    }
-
-    setMT5ExistingAccount(existing_account) {
-        this.mt5_existing_account = existing_account;
-    }
-
-    openFailedVerificationModal(selected_account_type) {
-        const {
-            common,
-            modules: { cfd },
-        } = this.root_store;
-        const { setJurisdictionSelectedShortcode, setAccountType } = cfd;
-        const { setAppstorePlatform } = common;
-
-        if (selected_account_type?.platform === CFD_PLATFORMS.MT5) {
-            setAppstorePlatform(selected_account_type.platform);
-            setAccountType({
-                category: selected_account_type.category,
-                type: selected_account_type.type,
-            });
-            this.setMT5ExistingAccount(selected_account_type);
-            setJurisdictionSelectedShortcode(selected_account_type.jurisdiction);
-        } else {
-            setJurisdictionSelectedShortcode('');
-        }
-        this.open_failed_verification_for =
-            selected_account_type?.platform === CFD_PLATFORMS.MT5 ? selected_account_type?.jurisdiction : 'multipliers';
-        this.toggleFailedVerificationModalVisibility();
+    toggleVerificationModal(value) {
```

Comment:
just a suggestion:
toggle means switching the value to opposite, right?
if we explicitly set the value, it would be more proper to call it something like setIsVerificationModalOpen